docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md

```diff
@@ -11,11 +11,13 @@ final Properties properties = new Properties();
 properties.setProperty(PropertyDefinition.PLUGINS.name, "readWriteSplitting,failover,efm");
 ```
 
-If you would like to use the read-write splitting plugin without the failover plugin, the Aurora host list plugin must be included before the read-write splitting plugin. This informs the driver that it should query for Aurora's topology.
+If you would like to use the read-write splitting plugin without the failover plugin, simply specify `readWriteSplitting` in the plugin chain.
+
+> NOTE: wrapper versions 2.2.3 and earlier required specifying the Aurora host list plugin to query Aurora's cluster topology like so: `"auroraHostList,readWriteSplitting"`.
```

Comment:
We could use a stronger wording here:

Since 2.2.3, the Aurora HostList Plugin is deprecated. Users using earlier versions still need to add the plugin to the plugin list. {Read more]

Reply:
no links yet, but we can link the release changelog later